src/core.ts

```diff
@@ -356,6 +358,8 @@ export type TrackingConsentDetails = {
   confirmed: boolean;
   /** Consent last-modified timestamp (ISO 8601) */
   timestamp: string;
+  /** Whether or not the UI has been shown to the end-user (undefined in older versions of airgap.js) */
+  isPrompted: boolean | undefined;
```

Comment:
the grammar here doesn't make sense. It reads as 'consent is prompted' (confusing). I'd much prefer a plain `prompted` field.

```suggestion
  prompted?: boolean;
```

Reply:
yeah good point, fixed!